### PR TITLE
Mkirk/fixup navbar

### PR DESCRIPTION
**Pull request by michaelkirk-signal**
Fixes some visual artifacts resulting from recent navbar changes.

PTAL @charlesmchen

### CODE REVIEW

File: Signal/src/ViewControllers/NewGroupViewController.m [hunk 1]
```diff
@@ -102,6 +102,8 @@ - (void)loadView
 
     self.title = [MessageStrings newGroupDefaultTitle];
 
+    self.view.backgroundColor = UIColor.whiteColor;
```
Comment (charlesmchen-signal) [inline Signal/src/ViewControllers/NewGroupViewController.m]: I suspect this should be:

```
    self.view.backgroundColor = Theme.backgroundColor;
```
Maybe I'll move that into `OWSViewController` so we can dry this up though.

Comment (charlesmchen-signal) [reply Signal/src/ViewControllers/NewGroupViewController.m]: Oh sorry, this is for v2.28. Forget it.

Comment (michaelkirk-signal) [reply Signal/src/ViewControllers/NewGroupViewController.m]: You're right - updated.

**Comment by michaelkirk-signal** [comment]
In 58e2e1383

**Pull request closed without merging**